vendor/github.com/elastic/go-structform/gotype/defs.go

```diff
@@ -20,7 +20,7 @@ package gotype
 import (
 	"reflect"
 
-	"github.com/elastic/go-structform"
+	structform "github.com/elastic/go-structform"
```

Comment:
Why are these aliases needed?

Reply:
Not really needed. It's goimports using aliases.